packages/astro/src/core/create-vite.ts

```diff
@@ -8,6 +8,7 @@ import astroVitePlugin from '../vite-plugin-astro/index.js';
 import astroPostprocessVitePlugin from '../vite-plugin-astro-postprocess/index.js';
 import markdownVitePlugin from '../vite-plugin-markdown/index.js';
 import jsxVitePlugin from '../vite-plugin-jsx/index.js';
+import resolveConfigPaths from './ssr/resolve-config-paths.js';
```

Comment:
```suggestion
import tsconfigAliasVitePlugin from './ssr/resolve-config-paths.js';
```

Reply:
Nit: Would `configAliasVitePlugin` be okay? As with our VSCode extension and the VSCode ecosystem out-of-the-box, both `tsconfig.json` and `jsconfig.json` are supported.

Reply:
Resolved in 222b5e169354df3bf62de7b875aff2846ee386a3